apis/userapi/userapi.yaml

```diff
@@ -1624,45 +1004,44 @@ paths:
             headers: {}
           x-testShouldPass: true
           x-testEnabled: true
-          x-testName: add skill to user
           x-testDescription: ''
       x-operation-settings:
         CollectParameters: false
         AllowDynamicQueryParameters: false
         AllowDynamicFormParameters: false
         IsMultiContentStreaming: false
-  /user/v1/skill/read:
+  /user/v1/unblock:
     post:
       description: >-
-        This API is used to read user added skill.
-           - The endpoint for **Get user skill** is `/user/v1/skill/read`
+        This API is used to unban/unblock a user
```

Comment:
@Hari-stackroute The word here should be 'remove the ban' or 'release' because unban is not a word.

Reply:
@Sunitashelar : done